datahub-web-react/src/app/search/AdvancedSearchFilter.tsx

```diff
@@ -0,0 +1,126 @@
+    }
+    border-top: 1px solid ${ANTD_GRAY[3]};
+`;
+
+const CloseSpan = styled.span`
+    :hover {
+        color: black;
+    }
+`;
+
+const StyledSearchFilterLabel = styled.div`
+    margin: 4px;
+`;
+
+const FilterFieldLabel = styled.span`
+    font-weight: 600;
+    margin-right: 2px;
+`;
+
+const TEXT_FILTERS = ['fieldPaths'];
+
+export const AdvancedSearchFilter = ({ facet, filter, onClose, onUpdate }: Props) => {
+    const [isEditing, setIsEditing] = useState(false);
+    return (
+        <>
+            <FilterContainer
+                onClick={() => {
+                    setIsEditing(!isEditing);
+                }}
+            >
+                <FieldFilterSection>
+                    <span>
+                        <FilterFieldLabel>{FIELD_TO_LABEL[filter.field]} </FilterFieldLabel>
+                        <AdvancedSearchFilterConditionSelect filter={filter} onUpdate={onUpdate} />
+                    </span>
+                    <CloseSpan
+                        role="button"
+                        onClick={(e) => {
+                            e.preventDefault();
+                            e.stopPropagation();
+                            onClose();
+                        }}
+                        tabIndex={0}
+                        onKeyPress={onClose}
+                    >
+                        <CloseOutlined />
+                    </CloseSpan>
+                </FieldFilterSection>
+                <ValueFilterSection>
+                    {TEXT_FILTERS.indexOf(filter.field) === -1 &&
+                        filter?.values?.map((value) => {
+                            const matchedAggregation = facet?.aggregations?.find(
+                                (aggregation) => aggregation.value === value,
+                            );
+                            if (!matchedAggregation) return null;
+
+                            return (
+                                <StyledSearchFilterLabel>
+                                    <SearchFilterLabel hideCount aggregation={matchedAggregation} field={value} />
+                                </StyledSearchFilterLabel>
+                            );
+                        })}
+                    {TEXT_FILTERS.indexOf(filter.field) !== -1 && filter?.values?.map((value) => <span>{value}</span>)}
+                </ValueFilterSection>
+            </FilterContainer>
+            {isEditing && (
+                <AdvancedFilterSelectValueModal
+                    facet={facet}
+                    onCloseModal={() => setIsEditing(false)}
+                    filterField={filter.field}
+                    onSelect={(values) => {
+                        const newFilter: FacetFilterInput = {
+                            field: filter.field,
+                            value: '',
+                            values: values as string[],
+                            condition: filter.condition,
+                            negated: filter.negated || false,
```

Comment:
won't this always be filter.negated? if it's optional you could just do `filter?.negated`